src/manage_nsfs/health.js

```diff
@@ -364,7 +369,8 @@ class NSFSHealth {
                 const config_file_path = this.config_fs.get_account_path_by_name(config_file_name);
                 res = await is_new_buckets_path_valid(config_file_path, config_data, storage_path);
             } else if (type === TYPES.BUCKET) {
-                res = await is_bucket_storage_path_exists(this.config_fs.fs_context, config_data, storage_path);
+                const account_fs_context = await this.get_account_owner_context(config_data.name, config_data.owner_account);
```

Comment:
can you move account_fs_context to be inside is_bucket_storage_and_owner_exists(), I don't see anymore usage of it outside, do I miss something?